cmd/gitbase/command/server.go

```diff
@@ -56,6 +56,7 @@ type Server struct {
 	DisableGit    bool `long:"no-git" description:"disable the load of git standard repositories."`
 	DisableSiva   bool `long:"no-siva" description:"disable the load of siva files."`
 	Verbose       bool `short:"v" description:"Activates the verbose mode"`
+	AltUast       bool `long:"alt-uast-serialization" description:"serialize uast in a new format" env:"GITBASE_UAST_SERIALIZATION"`
```

Comment:
could you change the flag to do the opposite? it should active old serialization format.